commit 4fcccde172e1d9655e56fba8f25b157a71da80e2
Author: Antti Palosaari <crope@iki.fi>
Date:   Mon Jan 16 19:27:41 2017 -0200

    cxd2820r: fix gpio null pointer dereference
    
    commit 0ffb94b6cc5df6376ab6bff5b80075641f6716f8 upstream.
    
    Setting GPIOs during probe causes null pointer deference when
    GPIOLIB was not selected by Kconfig. Initialize driver private
    field before calling set gpios.
    
    It is regressing bug since 4.9.
    
    Fixes: 07fdf7d9f19f ("[media] cxd2820r: add I2C driver bindings")
    
    Reported-by: Chris Rankin <rankincj@gmail.com>
    Tested-by: Chris Rankin <rankincj@gmail.com>
    Tested-by: Håkan Lennestål <hakan.lennestal@gmail.com>
    Signed-off-by: Antti Palosaari <crope@iki.fi>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1224_4fcccde172e1d9655e56fba8f25b157a71da80e2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1224_4fcccde172e1d9655e56fba8f25b157a71da80e2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
